superset/assets/visualizations/PlaySlider.css

```diff
@@ -0,0 +1,21 @@
+.play-slider {
+    height: 100px; 
```

Comment:
counldn't figure out how the DeckGL container knows how to set itself to `height() -100`

Reply:
That's the space currently left in the bottom, below the DeckGL container. TBH I wasn't really sure about the best way to place this.